docs/proposals/node-agent/README.md

```diff
@@ -0,0 +1,41 @@
+# Node Agent
+
+## Requirements
+
+### Which service must the domain provide
+
+The node agent amanges workload management. A workload is the tiniest unit in orka. It can be a container, a virtual machine, etc.
```

Comment:
```suggestion
The node agent manages `orka` workloads on a specific node. There is one agent per node.

* A node is a physical or virtual machine in the cluster, that is managed by the `orka` control plane.
* A workload is the smallest unit of compute deployable in orka. It can be e.g. a container, a virtual machine, etc.
```